File: ActuarialCalculators/AlgebraicLibrary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ActuarialCalculators
{
    public class AlgebraicLibrary
    {
        public static double Linear(double x)
        {
            return x;
        }

        public static double Cubic(double x)
        {
            return x * x * x;
        }

        public static double Quartic(double x)
        {
            return x * x * x * x;
        }

        public static double Quintic(double x)
        {
            return x * x * x * x * x;
        }

        public static double SquareRoot(double x)
        {
            return Math.Sqrt(x);
        }

        public static double CubeRoot(double x)
        {
            return Math.Cbrt(x);
        }
    }
}

File: Go/TheOpenMathLibrary/GeneralMathematics/AlgebraicLibrary.go
package generalmathematics

import (
	"math"
)

// AlgebraicLibrary provides various algebraic functions.
type AlgebraicLibrary struct{}

// Linear returns the linear value of x.
func (AlgebraicLibrary) Linear(x float64) float64 {
	return x
}

// Cubic returns the cubic value of x.
func (AlgebraicLibrary) Cubic(x float64) float64 {
	return x * x * x
}

// Quartic returns the quartic value of x.
func (AlgebraicLibrary) Quartic(x float64) float64 {
	return x * x * x * x
}

// Quintic returns the quintic value of x.
func (AlgebraicLibrary) Quintic(x float64) float64 {
	return x * x * x * x * x
}

// SquareRoot returns the square root of x.
func (AlgebraicLibrary) SquareRoot(x float64) float64 {
	return math.Sqrt(x)
}

// CubeRoot returns the cube root of x.
func (AlgebraicLibrary) CubeRoot(x float64) float64 {
	return math.Cbrt(x)
}
